src/main/java/org/usfirst/frc4904/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,83 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import org.usfirst.frc4904.standard.subsystems.SolenoidSubsystem;
+import org.usfirst.frc4904.standard.subsystems.SolenoidSubsystem.SolenoidState;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+
+public class Intake extends SubsystemBase {
+  private static final double defaultOnSpeed = 0.7;
+  private static final double defaultOffSpeed = 0.0;
+
+  protected final Motor intake, funnel, belt;
+  protected final SolenoidSubsystem solenoid;
+
+  /**
+   * Intake - Wraps the three intake SRXs (front intake, funnel, belt) and the
+   * intake solenoid
+   * 
+   * @param name           The name of the Intake
```

Comment:
Usually, we don't pass in a name to a year-specific thing, which we know will just be named "Intake", and which there is only one instance of.